ci-tests/datavolume/datavolume_test.go

```diff
@@ -50,18 +53,26 @@ var _ = AfterSuite(func() {
 })
 
 var _ = Describe("Data Volume Test", func() {
+	testName := "datavolume"
+	tfExecPath := "terraform"
+	It("init", func() {
+		if err := terraform.Init(testDir, testName, tfExecPath); err != nil {
+			Fail(fmt.Sprintf("failed to init terraform (runDir: %s, testName: %s, terraform path: %s) , with error: %s", testDir, testName, tfExecPath, err))
+		}
+	})
 	It("create", func() {
```

Comment:
terraform create

Reply:
Changed to `terraform apply (create new)`